src/sql/workbench/contrib/objectExplorer/browser/serverTreeView.ts

```diff
@@ -293,8 +293,9 @@ export class ServerTreeView extends Disposable implements IServerTreeView {
 			}
 			await this.refreshTree();
 			if (newProfile && !newProfileIsSelected) {
-				await this._tree!.reveal(newProfile);
-				this._tree.setFocus(newProfile);
+				await this._tree.reveal(newProfile);
+				await this._tree.select(newProfile);
```

Comment:
What are these changes fixing?

Reply:
@reverting back to old code from the last release